network/src/commonTest/kotlin/com/wire/kalium/api/base/authenticated/notification/AccessUpdateTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.wire.kalium.api.base.authenticated.notification
+
+import com.wire.kalium.model.EventContentDTOJson
+import com.wire.kalium.network.api.base.authenticated.conversation.model.ConversationAccessInfoDTO
+import com.wire.kalium.network.api.base.authenticated.notification.EventContentDTO
+import com.wire.kalium.network.api.base.model.ConversationAccessRoleDTO
+import com.wire.kalium.network.tools.KtxSerializer
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class AccessUpdateTest {
```

Comment:
Maybe we can add another test for the case if legacy `access_role` and `access_role_v2` are presented in the response

Reply:
If both fields are present in the response? That's I'm testing in `givenPayloadWithAccessRoleAndDeprecatedAccessRoleField_whenDecoding_thenDeprecatedFieldIsPreferred()`

Reply:
my bad did not notice it